cmd/cluster/openstack/create.go

```diff
@@ -170,6 +172,19 @@ func (o *RawCreateOptions) ApplyPlatformSpecifics(cluster *hyperv1.HostedCluster
 		cluster.Spec.Platform.OpenStack.IngressFloatingIP = o.OpenStackIngressFloatingIP
 	}
 
+	// If the user has specified DNS nameservers, it'll be used when creating the managed subnet(s).
+	// In the case where the user wants to override the other fields of the managed subnets, they can
+	// first render the cluster spec and then provide the needed configuration as API allows (for
+	// example the allocation pools and DNS nameservers).
```

Comment:
no need to mention DNS nameservers again as it will be possible to override with the CLI. No no need to render the manifest.